### PR TITLE
style: add missing markup for featured cards

### LINKED ISSUE #244: Bug: Featured cards are missing styles from the design
### Describe the bug

There are some contrast/legibility issues with the current layout. Some of the text is very small, and doesn't have a good `font-weight` to compensate. Some are calling too much attention for not having a subtler color.

#### Here are the differences in style:
- [x] Outer `border-radius` is `8px`. **Should be `16px`, or `1rem`.**
- [x] Organization name: 
  - [x] `font-size` is `.75rem`, or `12px`. **Should be `.875rem` (or `14px`).**
  - [x] `font-weight` is not set. **Should be `medium` (or `500`).**
- [x] Upvote button: 
  - [x] Color is black for the text, border, and icon. **Should be shades of gray.**
  - [x] **Background should be the lightest gray shade**, instead of white.
  - [x] #230 
- [x] Description text: has a `400` (regular) font-weight. **It should be `500`.**
- [ ] Data items (issues, stars, PRs, and contributors):
  - [x] `font-size` is `0.75rem` (`12px`). **Should be `0.875rem` (or `14px`).** It'll improve the legibility
  - [x] Color is black. **Should be shades of gray for the text and icon.**
  - [x] Icon sizes are set in `1em`. That means it'll be equal to 1 times the size of the parent `font-size`. In this case, it's `12px`. **It should be `1rem`, so it's based on the root `font-size`, which is `16px`.**
  - [ ] #286

**How it should look:**
Screen Shot 2022-07-27 at 12 04 41 PM

**How it currently looks:**
Screen Shot 2022-07-27 at 12 04 14 PM

### Steps to reproduce

[You can pick the CSS from Figma here.](https://www.figma.com/file/z0LEHrTwmvuCokhbYmIhPd/OpenSauced-%E2%80%94-Hot-(hot.opensauced.pizza)?node-id=3%3A2)

### Affected services

hot.opensauced.pizza

### Platforms

_No response_

### Browsers

_No response_

### Environment

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by shapaaa**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
This PR adds missing styles for featured cards to make them as per design in Figma

## Related Tickets & Documents

Fixes #244

## Mobile & Desktop Screenshots/Recordings
After merge the cards will have changes shown below

image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: tailwind.config.cjs
Comment (0-vortex) [inline tailwind.config.cjs:27]: the color codes could use better naming but LGTM! 👍

**Review by 0-vortex — APPROVED** [review]

File: tailwind.config.cjs
Comment (shapaaa) [reply tailwind.config.cjs:27]: Can you let me know how could I use better naming for variables?

Comment (bdougie) [reply tailwind.config.cjs:27]: Might be worth getting adding the variables we are using in the insights repo (currently private). https://github.com/open-sauced/hot/issues/275

Comment (0-vortex) [reply tailwind.config.cjs:27]: I'm thinking of connecting the variables to the current components/icons being treated, examples: `iconPullGrey`, `iconIssueFaded`, `hotIssueIconGrey`, etc

File: src/components/HotRepositories.tsx
Comment (NsdHSO) [inline src/components/HotRepositories.tsx:85]: If [this PR](https://github.com/open-sauced/hot/pull/269) merged before getting approval to do merge run `npm run format` "Hope it catches the withespaces"  if not please delete the empty spaces

**Comment by bdougie** [comment]
Rebase the main branch to get the latest on here. Looks like there are bugs on this PR that have been fixed already. https://github.com/open-sauced/hot/issues/261

**Pull request merged**
